### ISSUE TITLE
Undefined rule font-family-no-missing-generic-family-keyword

### ISSUE THREAD

**Issue by nextgenthemes** [AtomLinter/linter-stylelint#379]
`Undefined rule font-family-no-missing-generic-family-keyword`

I use the same config as bootstrap uses. https://github.com/twbs/bootstrap/blob/v4-dev/build/.stylelintrc I copied the file to the project root for this package to pick it up. I constantly have annoying popups with this error on every file save. I setup it this way that it will not lint when no config is found so it must be running with that config. I am not sure if this is part of the general linter updates to atom but what is particularly annoying it that its popups with the error rather then them showing up down in the linter messages when they can be hidden.

I installed all packages that they use with yarn. https://github.com/twbs/bootstrap/blob/v4-dev/package.json Running their lint task `"css-lint": "stylelint --config build/.stylelintrc --syntax scss \"scss/**/*.scss\"",` works perfectly fine, yet in atom (was not always the case, came up recently) I get this error all the time.

**Comment by nextgenthemes** [comment]
Just updated
```

stylelint-config-recommended-scss 2.0.0 to 3.0.0        
stylelint-scss                                    2.1.0 to 2.2.0
```

```
Now i get 

ENOENT: no such file or directory, open '/home/.../node_modules/stylelint-config-standard/node_modules/stylelint-config-recommended/index.js'
```

**Comment by Arcanemagus** [comment]
Fixed in #372.
